modules/ROOT/pages/fcct-config.adoc

```diff
@@ -0,0 +1,107 @@
+:experimental:
+= FCCT Configuration and Common Use Cases
+
+== Scope
+This document provides the configuration specification for human-readable FCCT YAML files. Once written, use fcct to convert the YAML file into the machine-readable JSON Ignition file.
+
+Fedora CoreOS (FCOS) Ignition files specify the configuration of FCOS. The process begins with a YAML configuration file. The FCOS Configuration Transpiler (FCCT) converts the human-readable YAML file into machine-readable JSON.
+
+== FCCT Configuration Specification
+
+Refer to the https://github.com/coreos/fcct/blob/master/docs/configuration-v1_0.md[Configuration Specification] for a full list of options.
+
+== Common Use Cases
+
+=== Running Containers
+
+Fedora CoreOS ships with both docker and podman installed. Use systemd units to start and stop containers.
+
+.Example for running busybox using systemd and podman
+[source,yaml]
+----
+variant: fcos
+version: 1.0.0
+systemd:
+  units:
+    - name: hello.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=MyApp
+        After=network-online.target
+        Wants=network-online.target
+
+        [Service]
+        TimeoutStartSec=0
+        ExecStartPre=-/bin/podman kill busybox1
+        ExecStartPre=-/bin/podman rm busybox1
+        ExecStartPre=/bin/podman pull busybox
+        ExecStart=/bin/podman run --name busybox1 busybox /bin/sh -c "trap 'exit 0' INT TERM; while true; do echo Hello World; sleep 1; done"
+
+        [Install]
+        WantedBy=multi-user.target
+----
+
+NOTE: When using docker instead of podman, units starting docker containers will need `Requires=docker.service` and `After=docker.service` to ensure the docker daemon is running before trying to start containers.
```

Comment:
I'd rather not mention `docker` here, as it has a different execution model compared to `podman` so the unit content could be quite different.